book/06-github/sections/2-contributing.asc

```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
+Ano pa, hahanapin ng GitHub ang mga listahan ng gawain sa iyong mga Isyu at Kahilingan na Pull at ipakita ang mga ito bilang metadata sa mga pahina na naglilista sa kanila. Halimbawa, kung mayroon kang Kahilingan na Pull sa mga gawain at titingnan mo ang pahina ng pangkalahatang-ideya ng lahat ng Kahilingan na Pull, maaari mong makita kung gaano kalayo ang ginawa nito. Tinutulungan nito ang mga tao na iwaksi ang Kahilingan na Pull sa mga subtask at tulungan ang ibang tao na subaybayan ang pag-unlad ng branch. Makikita mo ang halimbawa nito sa <<_task_list_progress>>.
 
 [[_task_list_progress]]
-.Task list summary in the Pull Request list.
+.Buod ng listahan ng gawain sa listahan ng Kahilingan na Pull.
 image::images/markdown-03-task-summary.png[Example Task List]
 
-These are incredibly useful when you open a Pull Request early and use it to track your progress through the implementation of the feature.
+Ang mga ito ay hindi kapani-paniwala na kapaki-pakinabang kapag binuksan mo ang isang Kahilingan na Pull nang maaga at gamitin ito upang subaybayan ang iyong pag-unlad sa pamamagitan ng pagpapatupad ng mga tampok.
 
-===== Code Snippets
+===== Mga Code Snippet
 
-You can also add code snippets to comments. This is especially useful if you want to present something that you _could_ try to do before actually implementing it as a commit on your branch. This is also often used to add example code of what is not working or what this Pull Request could implement.
+Maaari ka ring magdagdag ng mga code snippet sa mga komento. Ito ay lalong kapaki-pakinabang kung nais mong ipakita ang isang bagay na _maaari_ mong subukan na gawin bago aktwal na ipatupad ito bilang isang commit sa iyong branch. Ito ay kadalasang ginagamit upang magdagdag ng halimbawa ng code kung ano ang hindi gumagana o kung ano ang maaaring ipatupad ng Kahilingan na Pull.
 
-To add a snippet of code you have to ``fence'' it in backticks.
+Upang magdagdag ng isang snippet sa code, kailangan mong i-``fence'' ito sa mga backtick.
 
 [source,text]
 ----
 ```java
 for(int i=0 ; i < 5 ; i++)
 {
-   System.out.println("i is : " + i);
+   System.out.println("i ay : " + i);
 }
 ```
 ----
 
-If you add a language name like we did there with 'java', GitHub will also try to syntax highlight the snippet. In the case of the above example, it would end up rendering like <<_md_code>>.
+Kung nagdagdag ka ng isang pangalan ng wika tulad ng ginawa natin doon sa 'java', susubukan din ng GitHub na i-higlight ang syntax ng snippet. Sa kaso sa itaas na halimbawa, ito ay matatapos sa paggawa tulad ng <<_md_code>>.
 
```

Comment:
"GitHub na i-higlight ang syntax" change to "GitHub na i-highlight ang syntax" slight typo